gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/spring/ReactorConfiguration.java

```diff
@@ -127,6 +134,22 @@ public PlatformProcessorChainFactory globalProcessorChainFactory(
         );
     }
 
+    @Bean("httpRequestTimeout")
+    public Long httpRequestTimeout(
```

Comment:
See my previous comment about merging this and the grace timeout into a common bean

Reply:
done in https://github.com/gravitee-io/gravitee-api-management/pull/2224/commits/10c718e2cb8c3dcfb498664de8c2d42f9e3271f7